### ISSUE TITLE
model training

### ISSUE THREAD

**Issue by pang-yi-jie** [seriee/Multiresolution-HTC#4]
Can you expose the data configuration and the trained model, we are trying to reproduce your experiment, but we don't know what caused the model overfitting to be very bad (the training set performed well but the test set performed poorly)
